src/common.h

```diff
@@ -25,6 +25,8 @@
 
 #define ARRAYS_SIZE(arr) (sizeof(arr) / sizeof(arr[0]))
 
+#define MASK(n) (~(1 << n))
```

Comment:
Perhaps replacing "1" with "1U" to avoid unexpected signed integer overflow would be better?